### ISSUE TITLE
Computation of the test loss

### ISSUE THREAD

**Issue by loubnabnl** [lululxvi/deepxde#218]
Hi @lululxvi,

I solved a 1D heat equation using DeepXDE, but I get test loss vectors for which only the first value is not equal to zero. So I don't understand why since I assume we are using the same loss function as the training set. And this always gives a training loss greater than the test loss, which I find a bit odd.

![image](https://user-images.githubusercontent.com/44069155/107361975-726a9580-6ad8-11eb-9def-d8d92e2c63bc.png)
![image](https://user-images.githubusercontent.com/44069155/107362255-ce351e80-6ad8-11eb-839e-38ca3e33af9a.png)

Thank you.

**Comment by lululxvi** [comment]
In the test loss, I only keep the PDE loss, which is more important.

**Comment by thierryhen** [comment]
Hi @lululxvi,

Please can you explain what are the different numbers figuring in the training loss or test loss list?
